src/test/java/io/confluent/idesidecar/restapi/resources/KafkaConsumeResourceTest.java

```diff
@@ -357,82 +391,68 @@ void testConsumeRecordsProtoSchemaTopic() throws JsonProcessingException {
-        .post(
-            "/gateway/v1/clusters/%s/topics/%s/partitions/-/consume"
-                .formatted(KAFKA_CLUSTER_ID, SCHEMA_LESS_TOPIC_NAME)
-        )
-        .then()
-        .statusCode(200);
+    await()
+        .atMost(Duration.ofSeconds(10))
+        .untilAsserted(() -> {
+          given()
+              .when()
+              .contentType(ContentType.JSON)
+              .headers(CLUSTER_REQUEST_HEADERS)
+              .post(
+                  "/gateway/v1/clusters/%s/topics/%s/partitions/-/consume"
+                      .formatted(KAFKA_CLUSTER_ID, SCHEMA_LESS_TOPIC_NAME)
+              )
+              .then()
+              .statusCode(200);
+        });
   }
 
   @Test
   void testConsumeRecordsWithNonEmptyRequestBody() {
     setupSimpleConsumeApi();
-
-    given()
-        .when()
-        .contentType(ContentType.JSON)
-        .headers(CLUSTER_REQUEST_HEADERS)
-        .body("{\"from_beginning\": true,\"max_poll_records\":5}")
-        .post(
-            "/gateway/v1/clusters/%s/topics/%s/partitions/-/consume"
-                .formatted(KAFKA_CLUSTER_ID, SCHEMA_LESS_TOPIC_NAME)
-        )
-        .then()
-        .statusCode(200);
-
-    wireMock.verifyThat(
-        exactly(1),
-        postRequestedFor(urlEqualTo(
-            CCLOUD_SIMPLE_CONSUME_API_PATH.formatted(KAFKA_CLUSTER_ID, SCHEMA_LESS_TOPIC_NAME))
-        ).withRequestBody(equalToJson("{\"from_beginning\": true,\"max_poll_records\":5}"))
-    );
+    await()
+        .atMost(Duration.ofSeconds(10))
+        .untilAsserted(() -> {
+          given()
+              .when()
+              .contentType(ContentType.JSON)
+              .headers(CLUSTER_REQUEST_HEADERS)
+              .body("{\"from_beginning\": true,\"max_poll_records\":5}")
+              .post(
+                  "/gateway/v1/clusters/%s/topics/%s/partitions/-/consume"
+                      .formatted(KAFKA_CLUSTER_ID, SCHEMA_LESS_TOPIC_NAME)
+              )
+              .then()
+              .statusCode(200);
+
+          wireMock.verifyThat(
+              exactly(1),
+              postRequestedFor(urlEqualTo(
+                  CCLOUD_SIMPLE_CONSUME_API_PATH.formatted(KAFKA_CLUSTER_ID, SCHEMA_LESS_TOPIC_NAME))
+              ).withRequestBody(equalToJson("{\"from_beginning\": true,\"max_poll_records\":5}"))
+          );
+        });
   }
 
   @Test
   void testBadConsumeRequestThrows400() {
     setupSimpleConsumeApi();
-    given()
-        .when()
-        .contentType(ContentType.JSON)
-        .headers(CLUSTER_REQUEST_HEADERS)
-        .body("{\"faucets\": \"on\"}")
-        .post(
-            "/gateway/v1/clusters/%s/topics/%s/partitions/-/consume"
-                .formatted(KAFKA_CLUSTER_ID, SCHEMA_LESS_TOPIC_NAME)
-        )
-        .then()
-        .statusCode(400)
-        .body("attributeName", is("faucets"));
-  }
-
-  @Disabled  // Find a way to inject mock consumer for local kafka cluster
-  @Test
-  void testConsumeNoWorkyForConfluentLocal() {
```

Comment:
Consume does worky for Confluent Local now, has been for a long time. We should remove this test.